tests/filecheck/frontend/refactor/operations.py

```diff
@@ -0,0 +1,48 @@
+# RUN: python %s | filecheck %s
+
+from xdsl.dialects.arith import AddfOp, MulfOp, SubfOp
+from xdsl.dialects.bigint import AddOp, BigIntegerType, MulOp, SubOp
+from xdsl.dialects.builtin import Float64Type
+from xdsl.frontend.pyast.context import CodeContext
+from xdsl.frontend.pyast.program import FrontendProgram
+
+p1 = FrontendProgram()
+p1.register_type(int, BigIntegerType)
+p1.register_method(int, "__add__", AddOp)
```

Comment:
Let's aim for this API for now:
```suggestion
p1.register_method(int.__add__, AddOp)
```